src/functions/__tests__/casing.test.ts

```diff
@@ -1,23 +1,29 @@
-import { ICasingOptions } from '../../types';
+import { CasingType } from '../../types';
 import { casing } from '../casing';
 
-function runCasing(target: unknown, type: ICasingOptions['type'], disallowDigits?: boolean) {
+function runCasing(
+  target: unknown,
+  type: CasingType,
+  disallowDigits?: boolean,
+  separatorForCamelAndPascalCasing?: string,
```

Comment:
Maybe it's less about making specific "seperators for camel and pascal", and more... "I would like to be able to enforce kebab casing for this URL, but ofc the URL will have slashes as well as kebab."

Reply:
I've came up with a different approach where a new separator can be optionally define to separate "words".

ie. in `camel`, with separator `/`, allowing digits
**valid:**
- helloThere/iAm007
- h/elloThere/iAm007
- helloThere/iAm007

**invalid:**
- helloThere/iAm0/07
- hello/There/iAm007
- helloTherei/Am007

Thoughts?